### PR TITLE
Prefix local paths with {{ conf.PATH_PREFIX }}

### PR DESCRIPTION
**Pull request by eapearson**
This app runs at the path root `/applist/`. The pre-existing `PATH_PREFIX` config was previously applied this to the web app config. Now it is applied to local urls as well.

### CODE REVIEW

File: src/templates/base.html [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
+    {% endblock %}
+    {% block title %}
+    <title>KBase App Catalog</title>
+    {% endblock %}
+    <script src="{{ conf.ROOT_PREFIX }}/static/wp/js/jquery.js" type="text/javascript"></script>
+    <script src="{{ conf.ROOT_PREFIX }}/static/wp/js/jquery-migrate.min.js" type="text/javascript"></script>
+    <script src="{{ conf.ROOT_PREFIX }}/static/wp/js/kbaseUtils.js" type="text/javascript"></script>
+    <link crossorigin="anonymous" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css"
+        integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" rel="stylesheet">
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet" />
```
Comment (briehl) [inline src/templates/base.html:33]: Is this already included in the embedded Bootstrap, so we don't have to depend on the CDN?

**Comment by eapearson** [comment]
Checked the codebase, glyph icons are not even used. Just fontawesome. Removed the extra glyphicon load, didn't notice any difference nor any errors, etc.

**Comment by eapearson** [comment]
BTW one way to test this is to try to access the app at `http://localhost:5000/x`, see it fail, then update the config for ROOT_PREFIX to `/x`, then see it work. Then revert to `/applist`, see it fail, then access it at `http://localhost:5000/applist` and see it succeed.

**Comment by briehl** [comment]
Yup, did all the testing that way, and it works.

**Pull request merged**